### DISCUSSION TITLE
Upgraded from 500 to 700 z-stick: auto-discovered functionality has changed

### DISCUSSION THREAD

**Discussion by larstobi** [zwave-js/zwave-js-ui#3544] [Q&A] [answered]
I have upgraded from Aeotec 500 to 700 z-stick, and now zwave-js-ui has a yellowish symbol next to the controller node 1, and when I hover over it the text "Auto-discovered functionality has changed since the last interview, re-interview to apply". 

When I click on it, a pop-up box shows saying "Refresh info: Are you sure you want to re-interview node 1? All known information about your nodes will be discarded. Battery powered nodes need to be woken up, interaction with the nodes won't be reliable until the interview is done." And then there is a selection check-box that says "Reset security classes".

I'm not entirely sure about what this does. First off, what does "reset security classes" entail? And second, what new functionality do we expect to get?

Have any of you performed this before? What happened, and did you check the reset security classes box?

**Comment by robertsLando** [comment]
That checkbox simply tells you that the configuration file of that device has been updated (could be additional/fixed configuration values, compatibility flags, manuals etc...). In order to use it you need to re-interview the node and this api will be used: https://zwave-js.github.io/node-zwave-js/#/api/node?id=refreshinfo, the reset security classes is used to set the security classes to the defaults values (in case you changed them during the inclusion step)

**Reply by robertsLando** [reply]
cc @AlCalzone

**Comment by AlCalzone** [comment]
Eh, the controller shouldn't have this at all.
@robertsLando This is not about an end device!

**Reply by larstobi** [reply]
<img width="1686" alt="Skjermbilde 2024-01-22 kl  13 15 07" src="https://github.com/zwave-js/zwave-js-ui/assets/64238/72a24d06-9895-417f-989a-fdaaff5e07af">
<img width="631" alt="Skjermbilde 2024-01-22 kl  13 18 00" src="https://github.com/zwave-js/zwave-js-ui/assets/64238/8b26ad03-8cc0-4316-9bf3-6c1219169775">
<img width="1394" alt="Skjermbilde 2024-01-22 kl  13 18 23" src="https://github.com/zwave-js/zwave-js-ui/assets/64238/3248b25d-adf2-4300-a772-9d5914b7e45e">

**Reply by robertsLando** [reply]
oh I focused on the question and not the device, sorry! Yeah that's an issue, I see @AlCalzone opened one in his side so I will skip the fix on mine (that would have been to ignore that flag for controllers)

**Comment by AlCalzone** [comment, marked as answer]
https://github.com/zwave-js/node-zwave-js/pull/6625

**Reply by robertsLando** [reply]
👍🏼
